### PR TITLE
upgrade sbt

### PR DESCRIPTION
**Pull request by pjfanning**
https://github.com/apache/incubator-pekko-samples/issues/5

### CODE REVIEW

File: pekko-sample-distributed-workers-scala/project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.5.6
\ No newline at end of file
+sbt.version=1.8.2
```
Comment (mdedetrich) [inline pekko-sample-distributed-workers-scala/project/build.properties:1]: nit: newline

File: akka-sample-sharding-java/build.sbt [hunk 2]
```diff
@@ -42,7 +42,7 @@ lazy val `killrweather-fog` = project
   .in(file("killrweather-fog"))
   .settings(commonSettings)
   .settings(
-    mainClass in (Compile, run) := Some("sample.killrweather.fog.Fog"),
+    run / mainClass := Some("sample.killrweather.fog.Fog"),
```
Comment (mdedetrich) [inline akka-sample-sharding-java/build.sbt:45]: Shouldn't this be `Compile / run / mainClass`?

File: akka-sample-sharding-java/build.sbt [hunk 3]
```diff
@@ -27,7 +27,7 @@ lazy val killrweather = project
   .in(file("killrweather"))
   .settings(commonSettings)
   .settings(
-    mainClass in (Compile, run) := Some("sample.killrweather.KillrWeather"),
+    run / mainClass := Some("sample.killrweather.KillrWeather"),
```
Comment (mdedetrich) [inline akka-sample-sharding-java/build.sbt:30]: Shouldn't this be `Compile / run / mainClass`?

**Review by mdedetrich — CHANGES_REQUESTED** [review]
Found some issues

File: pekko-sample-distributed-workers-scala/project/build.properties (hunk 1 above)
Comment (pjfanning) [reply pekko-sample-distributed-workers-scala/project/build.properties]: added

File: akka-sample-sharding-java/build.sbt (hunk 3 above)
Comment (pjfanning) [reply akka-sample-sharding-java/build.sbt]: changed

**Comment by mdedetrich** [comment]
Cool, ill wait for CI to pass

**Review by mdedetrich — APPROVED** [review]
lgtm

**Pull request merged**